README.md

```diff
@@ -20,6 +20,22 @@ To use the latest release, add the following to your project:
 - [Word Count](https://github.com/FundingCircle/jackdaw/tree/master/examples/word-count)
 - [Simple Ledger](https://github.com/FundingCircle/jackdaw/tree/master/examples/simple-ledger)
 
+## Testing
+
+The included docker-compose file (adapted from confluent's ["all-in-one"](https://github.com/confluentinc/cp-docker-images/blob/5.1.0-post/examples/cp-all-in-one/docker-compose.yml) compose file)
+provides all the services required to run the project's tests. This allows the tests
+to be run as follows. You'll need up-to-date versions of docker, docker-compose and lein
+
+```
+docker-compose up -d
+docker run \
```

Comment:
This might not be a very friendly way to run tests imho.

Can't we just do like in other projects where we simply do something like
`lein with-profile docker test` for example, so you still run everything locally but simply connecting to the docker containers?